readthedocs/notifications/messages.py

```diff
@@ -29,8 +34,18 @@ def __repr__(self):
     def __str__(self):
         return f"Message: {self.id} | {self.header}"
 
+    def _escape_format_values(self, format_values):
+        """
+        Escape all potential HTML tags included in format values.
+
+        This is a protection against rendering potential values defined by the user.
+        It uses the Django's util function ``escape`` (similar to ``|escape`` template tag filter)
+        to convert HTML characters into regular characters.
+        """
+        return {key: escape(value) for key, value in format_values.items()}
```

Comment:
This means that nested values won't be supported. I think this should be fine, but probably worth documenting somewhere if it isn't already to avoid surprises.

Reply:
This is a good point. Currently, we don't need/support nested values so I think it's fine to write down a comment here for now 👍🏼

Reply:
Done in 24bd75d92